Potentiometer/CS/Package.appxmanifest

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="59b0fb39-8b10-4137-887e-f3dc3c9caaa1"
+    Publisher="CN=PotentiometerSensor"

```

Comment:
Change back to Publisher="CN=MSFT"
Change above line Name="59b0fb39-8b10-4137-887e-f3dc3c9caaa1"
to Name="PotentiometerSensor"

Reply:
Made the changes